Sources/Orbit/Support/Components/TimelineStep.swift

```diff
@@ -56,73 +71,119 @@ public struct TimelineStep: View {
                 Icon(style.iconSymbol, size: .large, color: style.color)
         }
     }
+}
 
-    var badgeStyle: Badge.Style {
-        switch style {
-            case .default:              return .neutral
-            case .status(let status):   return .status(status, inverted: false)
+// MARK: - Inits
+
+public extension TimelineStep where Header == TimelineStepBadgeText {
+
+    /// Creates Orbit TimelineStep component with custom details.
+    init(
+        _ label: String,
+        sublabel: String,
+        style: TimelineStepStyle,
```

Comment:
The subabel and style should have default values, same as other inits.

Reply:
I've removed it with intention. Without default values we have 4 different initializers which make it very clear how to init the component. With `defaults` values, we will have couple different one for almost each of them. Please compare:
![Screenshot 2022-07-22 at 11 30 44](https://user-images.githubusercontent.com/5494090/180410921-5f604916-8c04-41f8-9e4c-24f6cb7b3265.png)
![Screenshot 2022-07-22 at 11 34 03](https://user-images.githubusercontent.com/5494090/180410985-bde932eb-afeb-4243-af93-c65688dc2d01.png)

Reply:
The XCode 14 will greatly simplify this automatically, no need to force clients to fill empty values because of this. (Our other components work the same way)